metakb/main.py

```diff
@@ -63,6 +64,23 @@ def search(variation: Optional[str] = Query(None, description=v_description),
                         detail)
 
 
+@app.get('/api/v2/search/statements',
+         summary=search_summary,
+         response_description=search_response_description,
+         response_model=SearchStatementsService,
+         description=search_description,
```

Comment:
We should (either now or later) update descriptions to distinguish this endpoint from `/search`

Reply:
completely forgot about this. good catch. i also need to update schemas